README.md

```diff
@@ -20,11 +20,11 @@ yarn add --dev step-manager
 
 Online demo is available on [yoriiis.github.io/step-manager](https://yoriiis.github.io/step-manager)
 
-The project includes also examples of `StepManager` implementation in the directory `./examples/`.
+The project includes also the example of the `StepManager` implementation in the directory `./example/`.
```

Comment:
_The project also includes an example of an implementation of `StepManager` in the directory `./example/`._

Si je comprends bien, c'est l'exemple d'un type d'implémentation du StepManager ?

Reply:
Oui c'est un exemple d'implémentation du `StepManager`.
J'ai repris ta version.